Cerchie/app-open-sauced
# feat:add part of network prototype

## Description

This PR adds the part of this new network functionality:
 
![design-mockup](https://github.com/Cerchie/app-open-sauced/assets/54046179/09d7aade-9755-4484-90fe-a112fb3a124d)


## What type of PR is this? (check all applicable)

- [ X] 🍕 Feature

## Related Tickets & Documents

Related to: https://github.com/open-sauced-craftwork/Cerchie/issues/1

## Mobile & Desktop Screenshots/Recordings


https://github.com/Cerchie/app-open-sauced/assets/54046179/2e468dec-7a58-4de2-b18a-1c559ad7098a



## Steps to QA

Once fork is cloned and cd'ed:

1. `gh pr checkout 1 && npm install && npm run dev` 
2. Login (top right)
3. Navigate to `http://localhost:3000/hub/lists`
4. Create a new list with a few members if you haven't got any lists.
5. View your list.
6. Click the 'Connections' tab. 
7. Verify that the network graph is present and animated. 



## Added to documentation?

- [X ] 🙅 no documentation needed...yet

## [optional] Are there any post-deployment tasks we need to perform?
This commit should not be part of any deployment yet. 


## [optional] What gif best describes this PR or how it makes you feel?

![woman saying "I'm just getting warmed up"](https://github.com/Cerchie/app-open-sauced/assets/54046179/f4386829-b3b6-405f-b02d-7a1040a72d9c)
